src/commands/bgsave.json

```diff
@@ -19,11 +23,24 @@
         ],
         "arguments": [
             {
-                "name": "schedule",
-                "token": "SCHEDULE",
-                "type": "pure-token",
-                "optional": true,
-                "since": "3.2.2"
+                "name": "operation",
+                "type": "oneof",
+                "arguments": [
+                    {
+                        "name": "schedule",
+                        "token": "SCHEDULE",
+                        "type": "pure-token",
+                        "optional": true,
```

Comment:
Nice, but put optional on the oneof level instead.

The syntax is rendered from this and, although they're equivalent,

    BGSAVE [SCHEDULE | CANCEL]

looks better than

    BGSAVE <[SCEDULE] | CANCEL>

Reply:
done